node/rest/Cargo.toml

```diff
@@ -70,6 +70,10 @@ rev = "d170a9f"
 default-features = false
 optional = true
 
+[dependencies.snarkvm-ledger-puzzle-epoch]
```

Comment:
Ideally we expose this via `snarkvm::prelude` or some other reference to avoid having to reimport a snarkVM crate here. Especially one that will require updating the `rev`.